pkgs/servers/calibre-web/default.nix

```
@@ -7,13 +7,13 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "calibre-web";
-  version = "0.6.13";
+  version = "0.6.14";
 
   src = fetchFromGitHub {
     owner = "janeczku";
     repo = "calibre-web";
     rev = version;
-    sha256 = "sha256-zU7ujvFPi4UaaEglIK3YX3TJxBME35NEKKblnJRt0tM=";
+    sha256 = "sha256-rR5pUB3A0WNQxq7ZJ6ykua7hMlzs49aMmVbBUOkOVfA=";
   };
 
   prePatch = ''
```

```suggestion
  postPatch = ''
```

Do these still patch?

---

Everything still works when I do `postPatch`. But I already have a `postPatch` block here which moves a couple of files around to satisfy setuptools.

@SuperSandro2000 Is it better to merge `prePatch` with `postPatch`? If it is - why?

---

> Is it better to merge prePatch with postPatch?

yes

> If it is - why?

If a patch touches setup.cfg it would now fail to apply.